pkg/cidata/template.go

```diff
@@ -19,6 +19,15 @@ var templateFS embed.FS
 
 const templateFSRoot = "cidata.TEMPLATE.d"
 
+type CACerts struct {
+	RemoveDefaults bool
```

Comment:
This needs to be a pointer for the default processing to work properly:

```suggestion
	RemoveDefaults *bool
```

Reply:
Fixed.